server/sonar-web/src/main/js/apps/maintenance/main-view.js

```diff
@@ -34,7 +34,7 @@ export default Marionette.ItemView.extend({
     const that = this;
     this.requestOptions = {
       type: 'GET',
-      url: '/api/system/' + (this.options.setup ? 'db_migration_status' : 'status')
+      url: window.baseUrl + '/api/system/' + (this.options.setup ? 'db_migration_status' : 'status')
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert this concatenation to the use of a template. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3AS3512)